ui/src/components/sermon-notes/SermonNotesContainer.js

```diff
@@ -163,6 +164,15 @@ const SermonNotesContainer = (props) => {
     [editUserSermonNotes]
   );
 
+  useEffect(() => {
+    if (!user?.id) {
+      setLogin(false);
+    } else {
+      setLogin(true);
+    }
+    console.log(user?.id);
+  }, [user]);
+
```

Comment:
[minor]: the website will auto reload if there is a user, so u can just check directly whether user.id exist or not and show the button or not. This also works but i think its a bit overkill w useEffect lol.